waffle_hub/dataset/dataset.py

```diff
@@ -1643,9 +1644,15 @@ def export(self, data_type: Union[str, DataType]) -> str:
             str: exported dataset directory
         """
 
-        self._check_trainable()
```

Comment:
self._check_trainable() also exists in the split function `def split`. Please modify it as in `def export`.